include/dt-bindings/clock/imxrt1170-clock.h

```diff
@@ -47,20 +47,22 @@
 #define IMXRT1170_CLK_ROOT_LPUART1		33
 #define IMXRT1170_CLK_ROOT_ENET1		34
 #define IMXRT1170_CLK_ROOT_USDHC1		35
+#define IMXRT1170_CLK_ROOT_ADC1 		36
```

Comment:
Please keep the sorting order for the root clocks as per the reference manual. ADC1 and ADC2 registers are between SEMC and GPT1, so let's assign IDs 29 and 30 to the IMXRT1170_CLK_ROOT_ADC1 and ADC2